tools/kagami/src/main.rs

```diff
@@ -713,18 +717,44 @@ mod validator {
     impl<T: Write> RunArgs<T> for Args {
         fn run(self, writer: &mut BufWriter<T>) -> Outcome {
             writer
-                .write_all(&construct_validator()?)
+                .write_all(&construct_validator(None)?)
```

Comment:
Passing options as arguments is usually an anti-pattern. In this case, I'd use `dependency injection`, to move the logic up the call stack.